module/Frontpage/src/Module.php

```diff
@@ -48,8 +49,7 @@ public function getServiceConfig(): array
         return [
             'factories' => [
                 'frontpage_service_frontpage' => function (ContainerInterface $container) {
-                    $translator = $container->get('translator');
-                    $aclService = $container->get('decision_service_acl');
```

Comment:
Should not be removed?

Reply:
The `$aclService` I mean.

Reply:
I'm not sure why this went wrong while rebasing...Anyway, fixed it